book/src/getting_started.md

```diff
@@ -25,17 +40,79 @@ Rust compiler. Here are the system requirements (they're pretty modest):
 
 [ru]: https://www.rustup.rs/
 
-There are two ways to get started working with Amethyst:
+## Rust project basics
+
+**If you consider yourself a rust veteran you can probably skip this section,
+just create a binary project and add a dependency for amethyst from crates.io.**
+
+Once you have that setup you'll need to make a new cargo binary project for your
+game to be written in.  You can do this with the following command
+```
+$ cargo init --bin my_cool_game
+```
+Where my_cool_game is the name of your project.
+
+Next you'll want to add a dependency to amethyst.  To do this visit
```

Comment:
I think we can just use `amethyst = "*"`; that's what we have `Cargo.lock` after all.

Reply:
If you use that then I believe `cargo update` will retrieve breaking changes as well as patches.

Reply:
I guess that's the intended behavior for binaries.

Reply:
> I guess that's the intended behavior for binaries.
When you say "that" are you referring to "break when you run cargo update" or "don't break when you run cargo update"?

Reply:
The former; for binaries you'd just update specific packages after you checked if the new version is any use to you.